client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -60,11 +62,6 @@ export async function recordOrder( cart, orderId ) {
 		return;
 	}
 
-	if ( cart.total_cost < 0.01 ) {
```

Comment:
I don't understand how the removal of code in this file is related to GA setup. Could you provide more context?

Reply:
Reading the PR instructions, this might be unintended?

Reply:
Got it. It makes sense to have it here, then.